### ISSUE TITLE
Error thrown for 0 month old subjects

### ISSUE THREAD

**Issue by erikglee** [DCAN-Labs/BIBSnet#36]
### What happened?

For some reason I am getting an error when I try to run CABINET on subjects whose age is 0 months. When I change their age in the sessions tsv file, CABINET is able to run correctly. Based on conversations with @hough129 , this is likely because CABINET doesn't have a 0 month template.

### What command did you use?

```shell
singularity run -B /home/faird/shared/projects/HBCD_MIDB_IG/cabinet/cabinet_240_test_2/:/dir /home/midb-ig/shared/containers/leex6144/cabinet_2_4_0.sif /dir/bids /dir/output participant -end postbibsnet -jargs /home/cabinet/parameter-file-container.json
```

### What version of CABINET are you using?

2.4.0

### Relevant log output

```shell
Traceback (most recent call last):
  File "/home/cabinet/cabinet", line 1044, in <module>
    main()
  File "/home/cabinet/cabinet", line 69, in main
    json_args, sub_ses_IDs = get_params_from_JSON([get_stage_name(stg) for stg
  File "/home/cabinet/cabinet", line 211, in get_params_from_JSON
    return validate_cli_args(vars(parser.parse_args()), stage_names,
  File "/home/cabinet/cabinet", line 289, in validate_cli_args
    sub_ses_IDs[ix]["age_months"] = read_from_tsv(
  File "/home/cabinet/cabinet", line 519, in read_from_tsv
    desired_output = get_col_value_from_tsv(j_args, logger, tsv_path, ID_col, col_name, sub_ses)
  File "/home/cabinet/cabinet", line 546, in get_col_value_from_tsv
    print("ensure_prefixed: ", ensure_prefixed(sub_ses[1], "ses-") if ID_col == "session_id" else ensure_prefixed(sub_ses[0], "sub-"))
IndexError: tuple index out of range
```

**Comment by audreymhoughton** [comment]
Hi @erikglee, @rosemccollum tested this with 0 months in the `participants.tsv` and with the `--age` flag and could not reproduce the error. I am going to close this issue. Let me know if it happens again with the most recent versions of cabinet. It could be an issue with the column naming as well!

**Comment by audreymhoughton** [comment]
@erikglee 's subjects' age gets loaded in as a dtype of float64, where @rosemccollum 's was loaded in as a dtype of object. A float64 dtype must make it so that the "0" appears empty to the codebase. We need to force the TSV to load it in as an object for every TSV.

**Comment by rosemccollum** [comment]
The error was due to a logic error where `if not 0` evaluates to true which happens to raise an error. This is fixed in release 2.4.3.
